coffea/nanoevents/mapping/__init__.py

```diff
@@ -1,9 +1,18 @@
 from .uproot import TrivialUprootOpener, UprootSourceMapping
 from .parquet import TrivialParquetOpener, ParquetSourceMapping
-from .preloaded import SimplePreloadedColumnSource, PreloadedOpener, PreloadedSourceMapping
+from .preloaded import (
+    SimplePreloadedColumnSource,
+    PreloadedOpener,
+    PreloadedSourceMapping,
+)
 from .cached import CachedMapping
 
-__all__ = ['TrivialUprootOpener', 'UprootSourceMapping',
-           'TrivialParquetOpener', 'ParquetSourceMapping',
-           'SimplePreloadedColumnSource', 'PreloadedOpener', 'PreloadedSourceMapping'
-           'CachedMapping']
+__all__ = [
+    "TrivialUprootOpener",
+    "UprootSourceMapping",
+    "TrivialParquetOpener",
+    "ParquetSourceMapping",
+    "SimplePreloadedColumnSource",
+    "PreloadedOpener",
+    "PreloadedSourceMapping" "CachedMapping",
```

Comment:
I think this is missing a comma, so the two strings just get concatenated

Reply:
weird it doesn't cause any tests to fail?

Reply:
Perhaps none of the tests involve PreloadedSourceMapping or CachedMapping? That's a reason why this is a bad syntax rule, but it's a hold-over from C.

Reply:
This is a great example of how a code formatter can make certain bugs more obvious, since the missing comma was previously disguised by a line break.

Reply:
Yeah - it's certainly useful from that perspective. However both of those classes are tested! Anyway. I'll fix it up and start rc2.

Reply:
Specifying `__all__` in a module is kind of like overriding `__dir__` on a class: it doesn't actually change the behavior of getattr but does provide a (possibly restricted) list of "public" attributes